### ISSUE TITLE
Crashes after trying to open DevTools 

### ISSUE THREAD

**Issue by devjsy** [nwjs/nw.js#7245]
NWJS Version : 0.42.5 SDK
Operating System : Windows 10

NW crashes with exit code 0xC0000005 when 'move' event handler is attached and f12 is pressed to open DevTools

The following demonstrates the bug.
```
var win = gui.Window.get()
win.on("move", () => {})
win.showDevTools()
```

comment out the win.on("move", () => {}) and developer tool windows opens without crash

**Comment by rogerwang** [comment]
It works for me with the latest nightly build: https://dl.nwjs.io/live-build/nw43/20191210-032511/8f8b6a467/v0.43.0/

**Comment by devjsy** [comment]
Confirmed it is fixed in the new 0.43.0 release
